listenbrainz/db/user.py

```diff
@@ -104,7 +104,7 @@ def get_by_user_login_id(login_id):
         result = connection.execute(sqlalchemy.text("""
             SELECT {columns}
               FROM "user"
-             WHERE user_login_id = :user_login_id
+             WHERE login_id = :user_login_id
```

Comment:
with this change do we also want to change the name of this method? (get_by_user_login_id -> get_by_login_id)

Reply:
right, good point.

Reply:
Perfect